var/spack/repos/builtin/packages/libnotify/package.py

```diff
@@ -0,0 +1,51 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Libnotify(MesonPackage):
+    """libnotify is a library for sending desktop notifications"""
+
+    homepage = "https://github.com/GNOME/libnotify"
+    url      = "https://github.com/GNOME/libnotify/archive/0.7.9.tar.gz"
+
+    version('0.7.9', sha256='9bd4f5fa911d27567e7cc2d2d09d69356c16703c4e8d22c0b49a5c45651f3af0')
+
+    variant('docbook', default=False, description='Build docbook docs. Currently broken')
+    variant('tests', default=False, description='Build with tests. Currenlty broken')
+    variant('gtkdoc', default=False, description='Build with gtkdoc. Currently broken')
```

Comment:
In what way are these broken?

Reply:
The short answer is they're having trouble with finding the DTD and XSLTs for docbook. I left them in there in case someone wants to pick up that torch, but they can just as easily be removed.

Reply:
I'm indifferent, up to you

Reply:
I split the difference. Variants commented out with a note left as to why.